Anton_Kazlouski/06.Ansible.Cruise/roles/nginx/tasks/main.yaml

```diff
@@ -0,0 +1,59 @@
+---
+# Variable setup.
+- name: Include OS-specific variables.
+  include_vars: "{{ ansible_os_family }}.yaml"
+
+- name: Include Ubuntu deploy
+  include: nginx_ubuntu.yaml
+  when: ansible_distribution == 'Ubuntu'
+
+- name: vhost1.com addition
+  become: yes
+  become_method: sudo
+  lineinfile:
+    dest: "/etc/hosts"
+    regexp: '.*vhost1.com$'
+    line: '127.0.0.1	vhost1.com'
+    state: present
+
+- name: vhost2.com addition
+  become: yes
+  become_method: sudo
+  lineinfile:
+    dest: "/etc/hosts"
+    regexp: '.*vhost2.com$'
+    line: '127.0.0.1	vhost2.com'
+    state: present
+
+- name: Check connect
+  wait_for:
+    host: localhost
+    port: 80
+    state: started
+    delay: 0
+    timeout: 3
+
+- name: Check base url
+  uri:
+    url: http://localhost
+    return_content: yes
+  register: base_url
+
+- name: Check vhost1_url
```

Comment:
lines 42-58, could you please implement loop. there are a lot of lines, it can be shorter